src/actinia_core/resources/ephemeral_processing_with_export.py

```diff
@@ -209,11 +209,12 @@ def _export_raster(self, raster_name,
 
         The region of the raster layer can be used for export. In this case a temporary region
         will be used for export, so that the original region of the mapset is not modified.
+        COG-Driver: https://gdal.org/drivers/raster/cog.html
 
         Args:
             raster_name (str): The name of the raster layer
-            format (str): GTiff
-            additional_options (list): Unused
+            format (str): COG
+            additional_options (list): Additional GDAL create options
```

Comment:
It is still unused

Reply:
You are right, _export_raster is only used once and there is no additional options argument.